panel/scripts/settings.js

```diff
@@ -7,6 +7,9 @@ function Settings(repl) {
   this.repl = repl;
   this.DEFAULTS = this.data = {
     transformer: 'babel',
+    tabSize: 2,
+    indentUnit: 2,
+    indentWithTabs: true,
```

Comment:
I'd like to keep this defaulted to false (use spaces), as that's how it currently behaves